triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/dao/DbTriggerDao.scala

```diff
@@ -94,23 +94,19 @@ final class DbTriggerDao private (dataSource: DataSource with Closeable, xa: Con
 
   private def insertRunningTrigger(t: RunningTrigger): ConnectionIO[Unit] = {
     val insert: Fragment = sql"""
-        insert into running_triggers values (${t.triggerInstance}, ${t.triggerParty}, ${t.triggerName})
+        insert into running_triggers values (${t.triggerInstance}, ${t.triggerParty}, ${t.triggerName}, ${t.triggerToken})
       """
     insert.update.run.void
   }
 
   private def queryRunningTrigger(triggerInstance: UUID): ConnectionIO[Option[RunningTrigger]] = {
     val select: Fragment = sql"""
-        select (trigger_instance, trigger_name, trigger_party) from running_triggers
+        select trigger_instance, full_trigger_name, trigger_party, access_token from running_triggers
```

Comment:
I’m confused, how did this work before? The column name isn’t changed by this PR.

Reply:
It didn't work before, this code was untested so far. See https://github.com/digital-asset/daml/pull/7750#discussion_r508597331.